pgjdbc/src/main/java/org/postgresql/core/v3/SimpleParameterList.java

```diff
@@ -332,6 +332,10 @@ public String toString(@Positive int index, boolean standardConformingStrings) {
         type = "interval";
       } else if (paramType == Oid.NUMERIC) {
         type = "numeric";
+      } else if (paramType == Oid.UUID) {
```

Comment:
Don't we handle UUID above or is that only in some specific conditions?

If not, does everything in the above "handle in binary..." list need to be included here? (e.g. all the int types)